src/components/UserPasswordUpdate/UserPasswordUpdate.test.tsx

```diff
@@ -50,6 +57,7 @@ describe('Testing User Password Update', () => {
   const formData = {
     previousPassword: 'anshgoyal',
     newPassword: 'anshgoyalansh',
+    wrongPassword: 'akhil',
     confirmNewPassword: 'anshgoyalansh',
   };
```

Comment:
Instead of any names use something related to org.
Example: previosPassword=Palisadoes
newPassword= ThePalisadoesFoundation
wrongPassword="this is wrong password"

Reply:
Sure @noman2002